apps/src/templates/levelSummary/SubLevelDropdown.jsx

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {SimpleDropdown} from '@cdo/apps/componentLibrary/dropdown';
+import i18n from '@cdo/locale';
+
+const SubLevelDropdown = ({subLevels, handleChange}) => {
+  const subLevelNums = Array.from(
+    {length: subLevels.length},
+    (_, index) => index
+  );
+  return (
+    <SimpleDropdown
+      items={subLevelNums.map(subLevel => ({
+        value: subLevel,
+        text: i18n.question() + ': ' + (subLevel + 1),
+      }))}
+      onChange={handleChange}
+    />
+  );
+};
```

Comment:
Two options to simplify this:
1) https://lodash.com/docs/4.17.15#range instead of the `Array.from`
2) 
```suggestion
const SubLevelDropdown = ({subLevels, handleChange}) => {

  return (
    <SimpleDropdown
      items={subLevels.map((_, index) => ({
        value: index,
        text: i18n.question() + ': ' + (index + 1),
      }))}
      onChange={handleChange}
    />
  );
};
```

Reply:
Also, because this is just a simple wrapper around a SimpleDropdown, what do you think about moving the logic to `SummaryContainer` and deleting this file?

I like the intuition to make small files that contain bits of self-contained logic, but I don't know if there is enough logic here to need a whole new file. But completely up to you, I'm fine either way.

Reply:
Thanks for this suggestion on simplifying! Going to keep this as a separate file but I see your point and moving forward will definitely consider.